libs/sdk-ui/src/base/localization/bundles/en-US.json

```diff
@@ -1544,11 +1544,21 @@
         "comment": "Do not translate non-breakable space '{nbsp}'",
         "limit": 0
     },
-    "attributesDropdown.relevantValues.tooltip": {
+    "attributesDropdown.relevantValues.tooltip.parentFilters": {
         "value": "Available values for <strong>{child}</strong> are filtered by selected <strong>{parents}</strong>.",
         "comment": "Do not translate HTML markup enclosed in <> and {child}, {parents} placeholders as it will be replaced by list of filter names.",
         "limit": 0
     },
+    "attributesDropdown.relevantValues.tooltip.limits": {
+        "value": "Values for <strong>{child}</strong> are filtered by an additional filter setting. ",
+        "comment": "Do not translate HTML markup enclosed in <> and {child}, {parents} placeholders as it will be replaced by list of filter names.",
```

Comment:
there is no {parents} placeholder in this string

Reply:
Fixed, copy paste.